src/controllers/action-queue/WyreClient.js

```diff
@@ -41,6 +42,8 @@ export const makeWyreClient = async (opt: WyreClientOptions): Promise<WyreClient
     Accept: 'application/json'
   }
 
+  logyv(wyreSecret, 'wyreSecret')
```

Comment:
This looks like it crept in.

Reply:
Good catch, missed this when debugging for my fixes.